.github/workflows/azure-login-canary.yml

```diff
@@ -87,7 +87,7 @@ jobs:
               RUN_URL="https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID"
               REPORT="${REPORT}\r\n"
               if [ ${{needs.az-login-test.result}} == 'success' ]; then REPORT="${REPORT}\r\n|✅|<${RUN_URL}|az-login-test>"; else REPORT="${REPORT}\r\n|❌|<${RUN_URL}|az-login-test>"; fi
-              echo "::set-output name=report::$REPORT"
+              echo -e "report=$REPORT" >> $GITHUB_OUTPUT
```

Comment:
Have you tested this parameter "-e" ?

Reply:
Hi @YanaXu yes I tested it out, the '-e' parameter enables the echo command to honor the escape sequences like new line object [here](https://github.com/Azure/login/blob/master/.github/workflows/azure-login-canary.yml#L88). Please refer the example snippet below:-

<img width="411" alt="image" src="https://github.com/Azure/login/assets/60884268/8e76d4a8-f384-4cc9-af32-35c7af5272fb">

Reply:
Hi @cloudsky13 , I tried this workflow but it failed at "echo -e".

```yaml

name: Test Set Output

on:
  [workflow_dispatch,push]

jobs:
  testSetOutput:
    runs-on: ubuntu-latest    
    steps:
    
    - name: Set output variable
      id: set-output
      run: |
        TITLE="Login action OIDC flow tests update - "
        DATEVAR=`date "+%d/%m/%YT%H:%M:%S"`
        TITLE="${TITLE}${DATEVAR}"
        REPORT="${TITLE}\r\nLink to run - https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID\r\n"
        echo "----------------------------"
        echo "$REPORT"
        echo "----------------------------"
        echo -e "report=$REPORT" >> $GITHUB_OUTPUT
    - name: Get output variable
      run: |
        echo "${{steps.set-output.outputs.report}}"
```

Reply:
Hi @YanaXu , I tried this out and have removed the '-e' for my commit. There's no need for '-e' here since the escape sequences will be invoked when the output is called in the next stepfor slack notification. And for the test workflow you shared previously, it works perfectly with below change:-

`name: Test Set Output

on:
  [workflow_dispatch,push]

jobs:
  testSetOutput:
    runs-on: ubuntu-latest    
    steps:
    
    - name: Set output variable
      id: set-output
      run: |
        TITLE="Login action OIDC flow tests update - "
        DATEVAR=`date "+%d/%m/%YT%H:%M:%S"`
        TITLE="${TITLE}${DATEVAR}"
        REPORT="${TITLE}\r\nLink to run - https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID\n"
        echo "----------------------------"
        echo $REPORT
        echo "----------------------------"
        echo "report=$REPORT" >> $GITHUB_OUTPUT
        
    - name: Get output variable
      run: |
        echo **-e** "${{steps.set-output.outputs.report}}" `